README.md

```diff
@@ -8,29 +8,31 @@ This library builds on top of the [Microsoft.OpenAPI.NET](https://github.com/mic
 
 ## Getting started
 
+### Generating SDKs
+
+1. Install the required tools for your language. (refer to the [Supported Languages](#supported-languages) table under the **Required tools & dependencies** column)
+1. Intialize the target project and add the required dependencies. (refer to the [Supported Languages](#supported-languages) table under the **Required tools & dependencies** column)
+1. Get Kiota: clone the repository and [build Kiota locally](./docs/generator/build.md) **--or--** [download and run binaries](./docs/generator/binaries.md) **--or--** [install and run the dotnet tool](./docs/generator/tool.md) **--or--** [run the docker image](./docs/generator/docker.md).
```

Comment:
I would put the "clone the repo" as the -last- option. No one _really_ wants to do that. We all want the path of least resistance. That's probably one of the other three options.

maybe in order (trying to be lang agnostic here)
- Download binary
- Run via the docker image
- Dotnet tool
- clone repo

Reply:
```suggestion
1. Get Kiota: [download and run binaries](./docs/generator/binaries.md) **--or--** [run the docker image](./docs/generator/docker.md)  **--or--** [install and run the dotnet tool](./docs/generator/tool.md) **--or--** clone the repository and [build Kiota locally](./docs/generator/build.md).
```

Reply:
very good point, thanks!